src/v3/src/components/LoopbackProbe/LoopbackProbe.tsx

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import fetch from 'cross-fetch';
+import { FunctionComponent } from 'preact';
+import { useEffect } from 'preact/hooks';
+
+import Logger from '../../../../util/Logger';
+import { useOnSubmit } from '../../hooks';
+import { ActionParams, LoopbackProbeElement } from '../../types';
+import { isAndroid } from '../../util';
+
+type RequestOptions = {
+  url: string;
+  timeout: number;
+  method: 'GET' | 'POST';
+  data?: string;
+};
+
+/**
+ * Temporary request client can be removed if this functionality is added
+ * to auth-js library.
+ * @see https://oktainc.atlassian.net/browse/OKTA-561852
+ * @returns Promise<Response>
+ */
+const makeRequest = async ({
+  url, timeout, method, data,
+}: RequestOptions) => {
+  const controller = new AbortController();
```

Comment:
TODO; need to find an IE11 compatible way to handle abort, this is not available in IE11 and core-js polyfill does not have support for it currently either. Possibly use Promise.race? It still won't stop the request from completing but at least will keep the widget from hanging on long requests.

Reply:
added in [3abcdbb](https://github.com/okta/okta-signin-widget/pull/2979/commits/3abcdbb4a0e4d106fd146e06a90238f7722dbb23)